### ISSUE TITLE
Github

### ISSUE THREAD

**Issue by subbarajugv** [santhoshnimmala/beyoncloud#1]

**Comment by santhoshnimmala** [comment]
Git and Github is almost done editing is pending

**Comment by santhoshnimmala** [comment]
need to add Logo and animatios also

**Comment by bharathchandra7** [comment]
> Git and Github is almost done editing is pending

Yes, Editing is in Pending annaya.

**Comment by santhoshnimmala** [comment]
can we close this project @bharathchandra7  ?

**Comment by bharathchandra7** [comment]
> can we close this project @bharathchandra7 ?

Yes, I send all video links to VenkatRatnam. He Has to Recheck and give me feedback Then we will export it.
